design/fast_finality.md

```diff
@@ -0,0 +1,262 @@
+# Fast finality design
+
+## Scope
+
+The introduction of a single sequencer to improve cost-per-transaction during the bootstrapping phase, as 
+described in the [Bootstrapping Strategy design doc](./Bootstrapping_strategy.md).
+
+## Glossary
+
+* *Light batch*: A set of transactions considered to be soft-finalised
```

Comment:
I'd say a better definition is:
"a set of transactions produced by the designated sequencer, considered to be soft-finalised in that order"

Reply:
Fixed